frontend/src/container/AppLayout/index.tsx

```diff
@@ -331,7 +346,14 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 						<LayoutContent>
 							<ChildrenContainer
 								style={{
-									margin: isLogsView() || isTracesView() ? 0 : ' 0 1rem',
+									margin:
+										isLogsView() ||
+										isTracesView() ||
+										isDashboardView() ||
+										isDashboardWidgetView() ||
+										isDashboardListView()
+											? 0
+											: ' 0 1rem',
```

Comment:
not sure but check if this white space will cause any issue

Reply:
no it doesn't

Reply:
removed